sdk/iot/modelsrepository/HOW_TO_USE.md

```diff
@@ -0,0 +1,158 @@
+<!-- The following YAML bit is needed by the docs system to publish the samples online. Uncomment/Update it when the samples can be published publically -->
```

Comment:
How to use is the readme that was previously in the samples folder but removed because samples is now autogenerated. The autogenerated documentation isn't great so I've kept the readme I previous had in the samples folder as HOW_TO_USE.md

Reply:
/cc @willmtemple for feedback on the samples workflow